app/src/main/res/drawable/progress_bar.xml

```diff
@@ -9,7 +9,7 @@
         android:color="@color/component_color_shared_progress_bar_stroke_color" />
     </shape>
   </item>
-  <item android:id="@+id/progress">
+  <item>
```

Comment:
This comment is for the entire PR, can you add some screenshots/anything else through which it's easy to verify that changes before and after are working as intended?